modules/social_features/social_event/social_event.install

```diff
@@ -2111,29 +2117,36 @@ function social_event_update_10311(): string {
 /**
  * Add new field to Event node type; Add new message template.
  */
-function social_event_update_11401(): void {
-  $config_files = [
-    'core.entity_view_display.node.event.email_card' => 'core.entity_view_display.node.event.email_card_11401',
-    'message.template.user_was_enrolled_to_event' => 'message.template.user_was_enrolled_to_event_11401',
-  ];
+function social_event_update_11401() {
 
   $config_storage = \Drupal::service('config.storage');
   $config_path = \Drupal::service('extension.list.module')->getPath('social_event') . '/config/static';
   $source = new FileStorage($config_path);
 
-  foreach ($config_files as $origin_name => $update_name) {
-    $config_storage->write($origin_name, (array) $source->read($update_name));
-  }
+  $config_storage->write('message.template.user_was_enrolled_to_event', (array) $source->read('message.template.user_was_enrolled_to_event_11401'));
```

Comment:
this can be removed right?

Reply:
No because this needs to be created

Reply:
Wasn't the entire purpose of this to use the API though? and thus remove the config storage and rather use entity api related functions?

Reply:
Done